acvm/Cargo.toml

```diff
@@ -44,3 +44,7 @@ unstable-fallbacks = []
 [dev-dependencies]
 rand = "0.8.5"
 proptest = "1.2.0"
+blake2 = "0.10.6"
+sha2 = "0.10.6"
+sha3 = "0.10.6"
```

Comment:
Can you use the implementations which are exposed in the `blackbox_solvers` module?